hack/generator/pkg/codegen/code_generator.go

```diff
@@ -93,6 +94,9 @@ func (generator *CodeGenerator) Generate(ctx context.Context, outputFolder strin
 	// emit each package
 	klog.V(0).Infof("Writing output files into %v", outputFolder)
 	for _, pkg := range packages {
+		if ctx.Err() != nil {
```

Comment:
minor: worth annotating each of these requests with a `// check for cancellation`?

I ask just because `ctx.Err()` doesn't scream "cancellation" to me... although maybe that's just a failing of my go knowledge and this is standard go cancellation.